Lesson24-signup-form/assets/assets/ezgi/index.htm

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <div id="container">
+  <div class="form-container">
```

Comment:
You don't really need div element here if everything else is inside the form. However, you can keep the div if you decide to use the approach I shared below for moving h1, p, ul elements outside the form.